### ISSUE TITLE
file is empty

### ISSUE THREAD

**Issue by wangpengyu-debug** [eth-medical-ai-lab/Med-PRM#1]
the python/1_train_dataset_RAG_judge_labeling.py and scripts/1_train_dataset_RAG_judge_labeling.sh file is empty.

**Comment by jhyun0414** [comment]
Thank you for your interest in our work, @wangpengyu-debug  🙏
You're right — the files you mentioned are not yet uploaded.
The RAG_judge_labeling code is currently being revised and will be released in the near future.
In the meantime, the PRM training set, which is the result of running that script, is already available in the repository and can be used as-is.
We’ll update the repository as soon as the scripts are ready.

Thank you for your patience!

**Comment by ATP-BME** [comment]
Could you please provide the specific file path and name of the PRM training set? Is it in this repository? This will help me locate and access the data more efficiently.

**Comment by jw-sohn** [comment]
@ATP-BME
You can download the data directly from huggingface 
`https://huggingface.co/datasets/dmis-lab/llama-3.1-medprm-reward-training-set`
or run 
`python python/0_preparing.py`
by following readme

**Comment by ViperVille007** [comment]
@jw-sohn @jhyun0414 any update regarding releasing the codes?
